src/pins.h

```diff
@@ -21,4 +22,26 @@
 #define GPS_TX 34
 #endif
 
+#ifdef TTGO_T_Beam_Supreme
+#undef OLED_SDA
+#undef OLED_SCL
+#undef OLED_RST
+
+#define OLED_SDA 17
+#define OLED_SCL 18
+#define OLED_RST 16
+
+#define GPS_RX 9
+#define GPS_TX 8
+
+#define LORA_SCK       12
+#define LORA_MISO      13
+#define LORA_MOSI      11
+#define LORA_CS        10
+#define RADIO_DIO0_PIN -1
```

Comment:
According to my look into the schematics, the DIO0 pin is connected to GPIO 01.
@peterus does the `-1` have a special meaning?
```suggestion
#define RADIO_DIO0_PIN 1
```